### PR TITLE
10.0 docs

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

File: docs/how_to/export_contacts.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Export of contact is **not** done automatically by Odoo.
+
+Each user have to *bind* themselves on the contacts/customers he wants. This choice has been made not to sync automatically every created contact in Odoo to Exchange.
+
+
+### More explainations
+
+Logged as a user for whom contacts sync is enabled, simply go on the customers/contacts and create a contact.
+
+On the customer form, the user has to binf manually this contact to exchange. The process to *bind* is showed in the following screencast.
```
Comment (yvaucher) [inline docs/how_to/export_contacts.md]: binf?

In the context it might be `bind`

**Review by yvaucher — APPROVED** [review]
One typo spotted otherwise LGTM.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+![Logo](./images/OdooXchange.png) 
+
+## Principles
+
+Odoo Exchange Connector is a bi-directional connector, 100% compatible with Odoo 10.0 (Community and Enterprise edition) and Exchange 2010 SP2 version.
+
+Using this connector, you will be able to synchronize your contacts and your calendar events.
+
+Unlike other connectors existing to sync Odoo and Exchange, this connector makes a server to server connection. Nothing to setup in Outlook or other tools. Once configured, it works seemlessly to either import contact/calendar events created in Exchange or export contacts/calendar events created in Odoo.
```
Comment (lukebranch) [inline docs/README.md]: *seamlessly

**Comment by yvaucher** [comment]
@damdam-s could you also fix the link on readme that links to pyews lib ?

` [PyEWS library](https://github.com/camptocamp/PyEWS)`

https://github.com/camptocamp/PyEWS doesn't exist, or maybe you want to create the fork so it fixes it for all versions :wink:

**Comment by damdam-s** [comment]
@yvaucher We created a fork for PyEWS and I added the good branch inside.

**Review by yvaucher — APPROVED** [review]

**Pull request merged**